crates/context_aware_config/src/helpers.rs

```diff
@@ -310,6 +321,49 @@ pub fn add_config_version(
     Ok(version_id)
 }
 
+#[cfg(feature = "high-performance-mode")]
+pub async fn put_config_in_redis(
+    version_id: i64,
+    state: Data<AppState>,
+    tenant: Tenant,
+    db_conn: &mut PooledConnection<ConnectionManager<PgConnection>>,
+) -> superposition::Result<()> {
+    let config = generate_cac(db_conn)?;
+    let redis_config = serde_json::to_string(&json!(config)).map_err(|e| {
+        log::error!("failed to convert cac config to string: {}", e);
+        unexpected_error!("could not convert cac config to string")
+    })?;
+    let config_key = format!("{}::cac_config", *tenant);
+    let max_created_key = format!("{}::cac_config::max_created_at", *tenant);
+    let audit_id_key = format!("{}::cac_config::audit_id", *tenant);
+    let config_version_key = format!("{}::cac_config::config_version", *tenant);
+    let last_modified = DateTime::to_rfc2822(&Utc::now());
+    let _ = state
+        .redis
+        .set::<(), String, String>(config_key, redis_config, None, None, false)
+        .await;
+    let _ = state
+        .redis
+        .set::<(), String, String>(max_created_key, last_modified, None, None, false)
+        .await;
+    if let Ok(uuid) = event_log::event_log
+        .select(event_log::id)
+        .filter(event_log::table_name.eq("contexts"))
```

Comment:
Why just for contexts?
And is event log cache required in redis?

Reply:
Its not event log cache, it's just the audit_id. I borrowed this code from snapshotting function